src/doc.d

```diff
@@ -450,6 +451,10 @@ DDOC_PSYMBOL    = $(U $0)
 DDOC_PSUPER_SYMBOL = $(U $0)
 DDOC_KEYWORD    = $(B $0)
 DDOC_PARAM      = $(I $0)
+DDOC_CONSTRAINT      = $(NOTHING) $0
```

Comment:
Here you may use `$(SPACE)` as discussed above or if you don't want to define an extra symbol just rely on the empty expansion of the macro being defined:

```
DDOC_CONSTRAINT      = $(DDOC_CONSTRAINT) $0
```

The expansion inside the definition is guaranteed to expand to nothing.